deps/3rd/dep_create.sh

```diff
@@ -65,6 +65,9 @@ function get_os_release() {
       opensuse-leap)
         version_ge "15" && compat_centos7 && return
         ;;
+      arch)
+        compat_centos7 && return
```

Comment:
version check is needed

Reply:
Version check for arch is unnecessary and impossible since arch is a rolling-release system.

Reply:
Arch Linux use a release date as the os version.

Reply:
The release date of Arch Linux is the release date of the installation medium, not the os version.